src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -511,6 +518,21 @@ private void addNegativeHearingLengthsError(HearingTypeItem hearingTypeItem, Lis
         }
     }
 
+    private static LocalDateTime parseListedDate(String listedDate) {
+        if (isNullOrEmpty(listedDate)) {
+            return null;
+        }
+        try {
+            return LocalDateTime.parse(listedDate, OLD_DATE_TIME_PATTERN);
+        } catch (Exception e) {
+            try {
+                return LocalDateTime.parse(listedDate);
+            } catch (Exception ex) {
```

Comment:
do we want to log the exceptions in both catch blocks or just ignore them?

Reply:
<img width="1283" height="316" alt="Screenshot 2025-09-10 at 11 28 27" src="https://github.com/user-attachments/assets/1a77dc7e-eeec-4d05-8af0-b199c1d657eb" />

See the attached screenshot, in this case, the date returned will be just null, and you would never know what happened if you don't log the exception

Reply:
you are not logging the exception from initial parse either?